File: tests/fixtures/Sample.cs
using System;
using System.Collections.Generic;

namespace SampleApp
{
    /// <summary>
    /// User management service.
    /// </summary>
    public class UserService : BaseService, IUserService
    {
        private string _connectionString;
        public static readonly int MaxRetries = 3;

        /// <summary>
        /// Creates a new user in the system.
        /// </summary>
        public User CreateUser(string name, string email)
        {
            return new User();
        }

        public List<User> ListUsers(int limit)
        {
            return new List<User>();
        }

        protected bool ValidateInput(string input)
        {
            return true;
        }

        private void LoadCache()
        {
        }
    }

    public interface IUserService
    {
        User CreateUser(string name, string email);
    }

    public enum UserStatus
    {
        Active,
        Inactive,
        Suspended
    }
}

File: tests/fixtures/sample.swift
import Foundation
import UIKit

/// User management service.
public class UserService: BaseService, UserServiceProtocol {
    private var connectionString: String
    public static let maxRetries = 3

    /// Creates a new user in the system.
    public func createUser(name: String, email: String) -> User {
        return User()
    }

    public func listUsers(limit: Int) -> [User] {
        return []
    }

    internal func validateInput(_ input: String) -> Bool {
        return true
    }

    fileprivate func loadCache() {
    }

    init(connectionString: String) {
        self.connectionString = connectionString
    }
}

protocol UserServiceProtocol {
    func createUser(name: String, email: String) -> User
}

struct UserConfig {
    let maxRetries: Int
    let timeout: TimeInterval
}

extension UserService {
    func clearCache() {
    }
}

enum UserStatus {
    case active
    case inactive
    case suspended
}

func helperFunction(input: String) -> String {
    return input
}
